docs/monitors/statsd.md

```diff
@@ -0,0 +1,110 @@
+<!--- GENERATED BY gomplate from scripts/docs/monitor-page.md.tmpl --->
+
+# statsd
+
+
+
+### USAGE
+
+#### Adding dimensions to StatsD metrics
+
+The StatsD monitor can parse keywords from a statsd metric name by a set of
+converter that was configured by user.
+
+```
+converters:
+  - pattern: "cluster.cds_{traffic}_{mesh}_{service}-vn_{protocol}_{port}.{action}"
+    dimensions:
```

Comment:
This example is confusing, what is the point of the `protocol` and `port` sections here?  It is probably better to just send everything pulled out as a "section" as a dimension automatically without having to have a list.